examples/dishwasher-app/silabs/src/ElectricalSensorManager.cpp

```diff
@@ -0,0 +1,251 @@
+            ElectricalPowerMeasurement::OptionalAttributes::kOptionalAttributeApparentPower,
+            ElectricalPowerMeasurement::OptionalAttributes::kOptionalAttributeRMSVoltage,
+            ElectricalPowerMeasurement::OptionalAttributes::kOptionalAttributeRMSCurrent,
+            ElectricalPowerMeasurement::OptionalAttributes::kOptionalAttributeRMSPower,
+            ElectricalPowerMeasurement::OptionalAttributes::kOptionalAttributeFrequency,
+            ElectricalPowerMeasurement::OptionalAttributes::kOptionalAttributePowerFactor,
+            ElectricalPowerMeasurement::OptionalAttributes::kOptionalAttributeNeutralCurrent));
+
+    VerifyOrReturnError(gEPMInstance, CHIP_ERROR_NO_MEMORY, ChipLogError(AppServer, "Failed to allocate memory for EPM Instance");
+                        gEPMDelegate.reset());
+
+    // Register Attribute & Command handlers
+    err = gEPMInstance->Init();
+    VerifyOrReturnError(CHIP_NO_ERROR == err, err, ChipLogError(AppServer, "Init failed on gEPMInstance"); gEPMInstance.reset();
+                        gEPMDelegate.reset());
+
+    // Initialize EPM attributes
+    OperationalStateEnum state = GetDishwasherManager()->GetOperationalState();
+    UpdateEPMAllAttributes(state);
+
+    // Initialize EEM (Electrical Energy Management)
+    VerifyOrReturnError(!gEEMInstance, CHIP_ERROR_INCORRECT_STATE, ChipLogError(AppServer, "EEM Instance already exist"));
+
+    gEEMInstance = std::make_unique<ElectricalEnergyMeasurementInstance>(
+        EEMEndpointId, *gEPMDelegate,
+        BitMask<ElectricalEnergyMeasurement::Feature, uint32_t>(ElectricalEnergyMeasurement::Feature::kImportedEnergy,
+                                                                ElectricalEnergyMeasurement::Feature::kCumulativeEnergy),
+        BitMask<ElectricalEnergyMeasurement::OptionalAttributes, uint32_t>(
+            ElectricalEnergyMeasurement::OptionalAttributes::kOptionalAttributeCumulativeEnergyReset));
+
+    VerifyOrReturnError(gEEMInstance, CHIP_ERROR_NO_MEMORY, ChipLogError(AppServer, "Failed to allocate memory for EEM Instance"));
+
+    // Register Attribute & Command handlers
+    err = gEEMInstance->Init();
+    VerifyOrReturnError(CHIP_NO_ERROR == err, err, ChipLogError(AppServer, "Init failed on gEEMInstance"); gPTInstance.reset());
+
+    // Initialize PT (Power Topology)
+    VerifyOrReturnError(!gPTDelegate && !gPTInstance, CHIP_ERROR_INCORRECT_STATE,
+                        ChipLogError(AppServer, "PowerTopology Delegate or Instance already exist"));
+
+    gPTDelegate = std::make_unique<PowerTopologyDelegate>();
+    VerifyOrReturnError(gPTDelegate, CHIP_ERROR_NO_MEMORY, ChipLogError(AppServer, "Failed to allocate memory for PT Delegate"));
+
+    gPTInstance = std::make_unique<PowerTopologyInstance>(
+        PTEndpointId, *gPTDelegate,
+        BitMask<PowerTopology::Feature, uint32_t>(PowerTopology::Feature::kSetTopology, PowerTopology::Feature::kDynamicPowerFlow),
```

Comment:
Perhaps you don't need all of these features for a simple Dishwasher - you aren't generating power from the Dishwasher I hope?